### PR TITLE
Various SEO fixes

### PR DESCRIPTION
**Pull request by TheOtterlord**
- Fix astro-i18n-aut integration image
- Fix outbound links for multiple themes

### CODE REVIEW

File: src/content/themes/split.md [hunk 1]
```diff
@@ -18,12 +18,12 @@ tools:
 publishDate: "Jun 12, 2023"
 ---
 
-<p>A clean &amp; simple mobile app landing page with smooth animations &amp;&nbsp;<a href=\"https://lighthouse-metrics.com/lighthouse/checks/2a64498d-80d2-4b09-8a11-de038b4e52d7">perfect lighthouse scores</a>&nbsp;for improved SEO.</p>
+<p>A clean &amp; simple mobile app landing page with smooth animations &amp;&nbsp;<a href="https://lighthouse-metrics.com/lighthouse/checks/2a64498d-80d2-4b09-8a11-de038b4e52d7">perfect lighthouse scores</a>&nbsp;for improved SEO.</p>
 
-<p>Built using Astro,&nbsp;<a href=\"http://tailwindcss.com/">Tailwind CSS</a>&nbsp;&amp;&nbsp;<a href=\"http://typescriptlang.org/">TypeScript</a>.</p>
+<p>Built using Astro,&nbsp;<a href="http://tailwindcss.com/">Tailwind CSS</a>&nbsp;&amp;&nbsp;<a href="https://typescriptlang.org/">TypeScript</a>.</p>
```
Comment (ElianCodes) [inline src/content/themes/split.md:23]:
```suggestion
<p>Built using Astro,&nbsp;<a href="https://tailwindcss.com/">Tailwind CSS</a>&nbsp;&amp;&nbsp;<a href="https://typescriptlang.org/">TypeScript</a>.</p>
```

**Review by ElianCodes — COMMENTED** [review]
All good catches @TheOtterlord! I just found one `http` link

File: src/content/themes/split.md (hunk 1 above)
Comment (TheOtterlord) [reply src/content/themes/split.md:23]: aha, good catch!

**Review by FredKSchott — APPROVED** [review]
LGTM!

**Comment by FredKSchott** [comment]
merging so that I can re-format to fix linting errors on `main`, if there are any

**Pull request merged**